protocol/contracts/v2/templeLineOfCredit/TempleLineOfCredit.sol

```diff
@@ -343,6 +343,14 @@ contract TempleLineOfCredit is ITempleLineOfCredit, TempleElevatedAccess {
         emit LiquidationsPausedSet(isPaused);
     }
 
+    /**
+     * @notice Set the minimum amount of Temple which must be borrowed on each call.
+     */
+    function setMinBorrowAmount(uint128 amount) external override onlyElevatedAccess {
+        minBorrowAmount = amount;
```

Comment:
now that the value is not a constant, don't you need a min threshold to check against so it's not set below it?

Reply:
I don't think that's required - not a very strong likelihood and can always be corrected soon after